Dockerfile

```diff
@@ -19,5 +19,7 @@ COPY --from=app-stage ldes-server-application/target/ldes-server-application.jar
 COPY --from=app-stage ldes-server-infra-mongo/target/ldes-server-infra-mongo-jar-with-dependencies.jar ./plugins/
 COPY --from=app-stage ldes-server-port-ingestion-rest/target/ldes-server-port-ingestion-rest-jar-with-dependencies.jar ./plugins/
 COPY --from=app-stage ldes-server-port-fetch-rest/target/ldes-server-port-fetch-rest-jar-with-dependencies.jar ./plugins/
+COPY --from=app-stage ldes-fragmentisers/ldes-fragmentisers-geospatial/target/ldes-fragmentisers-geospatial-jar-with-dependencies.jar ./plugins/
+COPY --from=app-stage ldes-fragmentisers/ldes-fragmentisers-sequential/ldes-fragmentisers-sequential-timebased/target/ldes-fragmentisers-sequential-timebased-jar-with-dependencies.jar ./plugins/
```

Comment:
I got this comment from @sverholen where I could find myseld in: instead of "plugins" , to use "lib" since it's more of a standard name